jkube-kit/build/api/src/test/java/org/eclipse/jkube/kit/build/api/assembly/AssemblyConfigurationUtilsTest.java

```diff
@@ -27,29 +27,33 @@
 import org.eclipse.jkube.kit.common.AssemblyFileSet;
 import org.eclipse.jkube.kit.config.image.build.Arguments;
 import org.eclipse.jkube.kit.config.image.build.BuildConfiguration;
-
-import mockit.Expectations;
-import mockit.Injectable;
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
 
+import static org.mockito.Mockito.when;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.eclipse.jkube.kit.build.api.assembly.AssemblyConfigurationUtils.createDockerFileBuilder;
 import static org.eclipse.jkube.kit.build.api.assembly.AssemblyConfigurationUtils.getAssemblyConfigurationOrCreateDefault;
 import static org.eclipse.jkube.kit.build.api.assembly.AssemblyConfigurationUtils.getJKubeAssemblyFileSets;
 import static org.eclipse.jkube.kit.build.api.assembly.AssemblyConfigurationUtils.getJKubeAssemblyFiles;
 
+@RunWith(MockitoJUnitRunner.class)
 public class AssemblyConfigurationUtilsTest {
-
+  @Mock
+  BuildConfiguration buildConfiguration;
+  @Mock
+  Assembly assembly;
+  @Mock
+  AssemblyFile file;
+  @Mock
+  AssemblyFileSet fileSet;
```

Comment:
most of these mocks are used by a single test case, it's better if you manually initialize those for each test case, e.g.:
```java
Assembly assembly = mock(Assembly.class);
```